### PR TITLE
Python3 readiness migrations for range and objects

### PR DESCRIPTION
**Pull request by davidlange6**
migrations proposed by 

futurize -w --no-diffs -j 4 -f libfuturize.fixes.fix_object -f libfuturize.fixes.fix_xrange_w
ith_import `find cmssw -name \*.py`

which should in general help performance

### CODE REVIEW

File: Configuration/Geometry/scripts/generate2023Geometry.py [hunk 1]
```diff
@@ -1,179 +1,180 @@
...
-            print '\n'.join([anError for anError in errorList])
-            sys.exit(1)
-        else:
-            sys.exit(0)
-    else:
-        detectorTuple = tuple([aDict["abbrev"]+str(getattr(options,"v_"+aDict["name"])) for aDict in allDicts])
-        generateGeom(detectorTuple,options)
-        sys.exit(0)
-    
+from builtins import range
```
Comment (ianna) [inline Configuration/Geometry/scripts/generate2023Geometry.py]: Note, all these changes beneath are the white space changes.

**Comment by ianna** [comment]
+1

**Comment by prebello** [comment]
please test

File: HLTrigger/Configuration/python/Tools/confdb.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from builtins import object
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/Tools/confdb.py]: Why is this single import needed - without any other change? Is it overriding a default?

File: HLTrigger/HLTanalyzers/test/CreateCFGs.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/python
 #
 
+from builtins import range
```
Comment (Martin-Grunewald) [inline HLTrigger/HLTanalyzers/test/CreateCFGs.py]: Also here: Why is this single import needed - without any other change? Is it overriding a default?

**Comment by davidlange6** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Comment by ssekmen** [comment]
+1

**Comment by andrius-k** [comment]
+1

**Comment by prebello** [comment]
+1 

tests-approved

**Comment by alberto-sanchez** [comment]
+1

**Comment by Dr15Jones** [comment]
+1

**Comment by perrotta** [comment]
Ah, ok: Thank you David and Chris!
Ready to sign for reco as soon as conflicts are fixed

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by davidlange6** [comment]
ok - i guess my prs are interfering:)

**Comment by fabiocos** [comment]
+operations

changes to Configuration packages consistent with the declared purpose of the PR

**Comment by fabiocos** [comment]
@davidlange6 unfortunately yes, there are conflicts. May be it is better to try to merge this PR quickly and then continue with the print.

@kpedro88 @ggovi @lpernie @arunhep @rekovic @thomreis please check asap and sign, in any case I want to close this migration by the week-end, unless you have real objections

**Comment by kpedro88** [comment]
+upgrade
(assuming conflicts get resolved)

**Comment by fabiocos** [comment]
@davidlange6 please let me know whether you prefer to give priority to this PR and keep on hold the other for print, or viceversa

**Comment by fabiocos** [comment]
@davidlange6 ok, so let's give priority to those

**Comment by davidlange6** [comment]
will remake after print migration

**Pull request closed without merging**